emails/mod-locked

```diff
@@ -6,10 +6,8 @@ Unfortunately, your mod {{mod.name}} ({{url}}) has been locked and unpublished b
 
 
 Please contact us by replying to this e-mail, or via one of the following channels:
-
-{{#support_channels}}
-  * {{name}}: {{channel_url}}
-{{/support_channels}}
+{% for name, channel_url in support_channels.items() %}
+  * {{name}}: {{channel_url}}{% endfor %}
```

Comment:
```suggestion
  * {{name}}: {{channel_url}}
{% endfor %}
```

I think it's a little easier to read if the `endfor` has its own line

Reply:
> The - will prevent an extra newline from appearing.

Thanks! Didn't know about that!